patches/@svgr+core+5.5.0.patch

```diff
@@ -0,0 +1,107 @@
+diff --git a/node_modules/@svgr/core/index.d.ts b/node_modules/@svgr/core/index.d.ts
+new file mode 100644
+index 0000000..7b54499
+--- /dev/null
++++ b/node_modules/@svgr/core/index.d.ts
+@@ -0,0 +1,101 @@
++export interface TemplateOptions extends SvgrOpts {}
```

Comment:
why are you patching these package typings?

Reply:
Since the plugins for esbuild are written in TypeScript to use these modules we need a typings files - I couldn't get it to work with adding them directly into the repo so I thought it easier to patch them into node_modules for typecheck to work.